Analysis/Tasks/PWGHF/HFDplusToPiKPiCandidateSelector.cxx

```diff
@@ -125,7 +125,7 @@ struct HFDplusToPiKPiCandidateSelector {
     if (hfCandProng3.cpaXY() < cuts[pTBin][6]) {
       return false;
     }
-    if (hfCandProng3.maxNormalisedDeltaIP() > cuts[pTBin][7]) {
+    if (TMath::Abs(hfCandProng3.maxNormalisedDeltaIP()) > cuts[pTBin][7]) {
```

Comment:
Why not use `std::abs` directly instead of calling in unnecessarily from a ROOT header?

Reply:
At the moment, in selectors, we are using the `ROOT::TMath` library. E.g. https://github.com/AliceO2Group/AliceO2/blob/485d56d55172c286cdcfce47f5106b52f0e19dfc/Analysis/Tasks/PWGHF/HFD0CandidateSelector.cxx#L183-L193 
or 
https://github.com/AliceO2Group/AliceO2/blob/485d56d55172c286cdcfce47f5106b52f0e19dfc/Analysis/Tasks/PWGHF/HFLcCandidateSelector.cxx#L145-L151
I stuck to this for consistency.

Reply:
It is consistent but overcomplicating it.
But OK, if you prefer, I will fix it centrally.

Reply:
Thanks, @vkucera. In the end, it is a matter of conventions. I don't think it makes any difference from the performance point of view.